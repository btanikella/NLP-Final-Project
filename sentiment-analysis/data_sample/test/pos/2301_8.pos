IN DT NN IN DT NNS VBZ DT RB VBN  JJ  CC RB VBD NN
PRP VBZ DT NN NN IN DT NN WP VBZ PRP$ NN IN VBG IN NN IN DT NN
DT NN IN PRP$ NN VBZ TO VB PRP$ NN IN PRP$ NN
DT NN VBZ RB JJ  NN JJ MD RB VB PRP RP
