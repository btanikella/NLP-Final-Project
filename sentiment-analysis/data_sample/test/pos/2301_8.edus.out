Sentence #1 (18 tokens):
In the Realm of the Senses is a beautifully filmed, well-written, and splendidly acted film.
[Text=In CharacterOffsetBegin=4 CharacterOffsetEnd=6 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=7 CharacterOffsetEnd=10 PartOfSpeech=DT] [Text=Realm CharacterOffsetBegin=11 CharacterOffsetEnd=16 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=17 CharacterOffsetEnd=19 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=20 CharacterOffsetEnd=23 PartOfSpeech=DT] [Text=Senses CharacterOffsetBegin=24 CharacterOffsetEnd=30 PartOfSpeech=NNS] [Text=is CharacterOffsetBegin=31 CharacterOffsetEnd=33 PartOfSpeech=VBZ] [Text=a CharacterOffsetBegin=34 CharacterOffsetEnd=35 PartOfSpeech=DT] [Text=beautifully CharacterOffsetBegin=36 CharacterOffsetEnd=47 PartOfSpeech=RB] [Text=filmed CharacterOffsetBegin=48 CharacterOffsetEnd=54 PartOfSpeech=VBN] [Text=, CharacterOffsetBegin=54 CharacterOffsetEnd=55 PartOfSpeech=,] [Text=well-written CharacterOffsetBegin=56 CharacterOffsetEnd=68 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=68 CharacterOffsetEnd=69 PartOfSpeech=,] [Text=and CharacterOffsetBegin=70 CharacterOffsetEnd=73 PartOfSpeech=CC] [Text=splendidly CharacterOffsetBegin=74 CharacterOffsetEnd=84 PartOfSpeech=RB] [Text=acted CharacterOffsetBegin=85 CharacterOffsetEnd=90 PartOfSpeech=VBD] [Text=film CharacterOffsetBegin=91 CharacterOffsetEnd=95 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=95 CharacterOffsetEnd=96 PartOfSpeech=.] 
root(ROOT-0, is-7)
prep(is-7, In-1)
det(Realm-3, the-2)
pobj(In-1, Realm-3)
prep(Realm-3, of-4)
det(Senses-6, the-5)
pobj(of-4, Senses-6)
dep(is-7, a-8)
advmod(filmed-10, beautifully-9)
dep(a-8, filmed-10)
punct(filmed-10, ,-11)
conj(filmed-10, well-written-12)
punct(filmed-10, ,-13)
cc(filmed-10, and-14)
advmod(acted-16, splendidly-15)
conj(filmed-10, acted-16)
dobj(acted-16, film-17)
punct(is-7, .-18)

Sentence #2 (20 tokens):
It tells the haunting story of a woman who kills her husband after falling in love with another man.
[Text=It CharacterOffsetBegin=101 CharacterOffsetEnd=103 PartOfSpeech=PRP] [Text=tells CharacterOffsetBegin=104 CharacterOffsetEnd=109 PartOfSpeech=VBZ] [Text=the CharacterOffsetBegin=110 CharacterOffsetEnd=113 PartOfSpeech=DT] [Text=haunting CharacterOffsetBegin=114 CharacterOffsetEnd=122 PartOfSpeech=NN] [Text=story CharacterOffsetBegin=123 CharacterOffsetEnd=128 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=129 CharacterOffsetEnd=131 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=132 CharacterOffsetEnd=133 PartOfSpeech=DT] [Text=woman CharacterOffsetBegin=134 CharacterOffsetEnd=139 PartOfSpeech=NN] [Text=who CharacterOffsetBegin=140 CharacterOffsetEnd=143 PartOfSpeech=WP] [Text=kills CharacterOffsetBegin=144 CharacterOffsetEnd=149 PartOfSpeech=VBZ] [Text=her CharacterOffsetBegin=150 CharacterOffsetEnd=153 PartOfSpeech=PRP$] [Text=husband CharacterOffsetBegin=154 CharacterOffsetEnd=161 PartOfSpeech=NN] [Text=after CharacterOffsetBegin=162 CharacterOffsetEnd=167 PartOfSpeech=IN] [Text=falling CharacterOffsetBegin=168 CharacterOffsetEnd=175 PartOfSpeech=VBG] [Text=in CharacterOffsetBegin=176 CharacterOffsetEnd=178 PartOfSpeech=IN] [Text=love CharacterOffsetBegin=179 CharacterOffsetEnd=183 PartOfSpeech=NN] [Text=with CharacterOffsetBegin=184 CharacterOffsetEnd=188 PartOfSpeech=IN] [Text=another CharacterOffsetBegin=189 CharacterOffsetEnd=196 PartOfSpeech=DT] [Text=man CharacterOffsetBegin=197 CharacterOffsetEnd=200 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=200 CharacterOffsetEnd=201 PartOfSpeech=.] 
root(ROOT-0, tells-2)
nsubj(tells-2, It-1)
det(story-5, the-3)
nn(story-5, haunting-4)
dobj(tells-2, story-5)
prep(story-5, of-6)
det(woman-8, a-7)
pobj(of-6, woman-8)
nsubj(kills-10, who-9)
rcmod(woman-8, kills-10)
poss(husband-12, her-11)
dobj(kills-10, husband-12)
prep(kills-10, after-13)
pcomp(after-13, falling-14)
prep(falling-14, in-15)
pobj(in-15, love-16)
prep(love-16, with-17)
det(man-19, another-18)
pobj(with-17, man-19)
punct(tells-2, .-20)

Sentence #3 (14 tokens):
The ghost of her husband continues to haunt her lond after his murder.
[Text=The CharacterOffsetBegin=206 CharacterOffsetEnd=209 PartOfSpeech=DT] [Text=ghost CharacterOffsetBegin=210 CharacterOffsetEnd=215 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=216 CharacterOffsetEnd=218 PartOfSpeech=IN] [Text=her CharacterOffsetBegin=219 CharacterOffsetEnd=222 PartOfSpeech=PRP$] [Text=husband CharacterOffsetBegin=223 CharacterOffsetEnd=230 PartOfSpeech=NN] [Text=continues CharacterOffsetBegin=231 CharacterOffsetEnd=240 PartOfSpeech=VBZ] [Text=to CharacterOffsetBegin=241 CharacterOffsetEnd=243 PartOfSpeech=TO] [Text=haunt CharacterOffsetBegin=244 CharacterOffsetEnd=249 PartOfSpeech=VB] [Text=her CharacterOffsetBegin=250 CharacterOffsetEnd=253 PartOfSpeech=PRP$] [Text=lond CharacterOffsetBegin=254 CharacterOffsetEnd=258 PartOfSpeech=NN] [Text=after CharacterOffsetBegin=259 CharacterOffsetEnd=264 PartOfSpeech=IN] [Text=his CharacterOffsetBegin=265 CharacterOffsetEnd=268 PartOfSpeech=PRP$] [Text=murder CharacterOffsetBegin=269 CharacterOffsetEnd=275 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=275 CharacterOffsetEnd=276 PartOfSpeech=.] 
root(ROOT-0, continues-6)
det(ghost-2, The-1)
nsubj(continues-6, ghost-2)
prep(ghost-2, of-3)
poss(husband-5, her-4)
pobj(of-3, husband-5)
aux(haunt-8, to-7)
xcomp(continues-6, haunt-8)
poss(lond-10, her-9)
dobj(haunt-8, lond-10)
prep(haunt-8, after-11)
poss(murder-13, his-12)
pobj(after-11, murder-13)
punct(continues-6, .-14)

Sentence #4 (14 tokens):
This film is really good, anyone interested should definitely check it out.
[Text=This CharacterOffsetBegin=281 CharacterOffsetEnd=285 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=286 CharacterOffsetEnd=290 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=291 CharacterOffsetEnd=293 PartOfSpeech=VBZ] [Text=really CharacterOffsetBegin=294 CharacterOffsetEnd=300 PartOfSpeech=RB] [Text=good CharacterOffsetBegin=301 CharacterOffsetEnd=305 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=305 CharacterOffsetEnd=306 PartOfSpeech=,] [Text=anyone CharacterOffsetBegin=307 CharacterOffsetEnd=313 PartOfSpeech=NN] [Text=interested CharacterOffsetBegin=314 CharacterOffsetEnd=324 PartOfSpeech=JJ] [Text=should CharacterOffsetBegin=325 CharacterOffsetEnd=331 PartOfSpeech=MD] [Text=definitely CharacterOffsetBegin=332 CharacterOffsetEnd=342 PartOfSpeech=RB] [Text=check CharacterOffsetBegin=343 CharacterOffsetEnd=348 PartOfSpeech=VB] [Text=it CharacterOffsetBegin=349 CharacterOffsetEnd=351 PartOfSpeech=PRP] [Text=out CharacterOffsetBegin=352 CharacterOffsetEnd=355 PartOfSpeech=RP] [Text=. CharacterOffsetBegin=355 CharacterOffsetEnd=356 PartOfSpeech=.] 
root(ROOT-0, good-5)
det(film-2, This-1)
nsubj(good-5, film-2)
cop(good-5, is-3)
advmod(good-5, really-4)
punct(good-5, ,-6)
nsubj(check-11, anyone-7)
amod(anyone-7, interested-8)
aux(check-11, should-9)
advmod(check-11, definitely-10)
ccomp(good-5, check-11)
dobj(check-11, it-12)
prt(check-11, out-13)
punct(good-5, .-14)

